_integration-tests/tests/vault/main.go

```diff
@@ -0,0 +1,71 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+package vault
```

Comment:
I think you are missing running the `go generate ./tests` so this test is actually added to the test suite.

Reply:
You are right. I probably missed a `go generate ./...`.